src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -92,32 +279,37 @@ class EmojiPickerMenu extends Component {
      * so that the sticky headers function properly
      *
      * @param {Object} item
+     * @param {Number} index
      * @returns {*}
      */
-    renderItem({item}) {
-        if (item.code === CONST.EMOJI_SPACER) {
+    renderItem({item, index}) {
+        const {code, header} = item;
+        if (code === CONST.EMOJI_SPACER) {
             return null;
         }
 
-        if (item.header) {
+        if (header) {
             return (
                 <Text style={styles.emojiHeaderStyle}>
-                    {item.code}
+                    {code}
                 </Text>
             );
         }
 
         return (
             <EmojiPickerMenuItem
                 onPress={this.props.onEmojiSelected}
-                emoji={item.code}
+                onHover={() => this.setState({highlightedIndex: index})}
+                emoji={code}
+                isHighlighted={index === this.state.highlightedIndex}
             />
         );
     }
 
     render() {
+        const pointerEvents = this.state.shouldDisablePointerEvents ? 'none' : 'auto';
```

Comment:
What are the pointerEvents?

Reply:
It's essentially a string passed into the props for a `View` directly underneath. It basically disables the ability for the View to receive pointer events (in this case hover events). You can think of this as the `pointerEvents` css property, which is either `none` or `auto` depending on whether you want them to be enabled or not. I can update the name to `pointerEventsStr` to clarify things if you think that's better!

Reply:
Oh you didn't have to change it, I simply didn't know what they were hehe Maybe a comment would have been enough, I am just trying to learn more how things in React Native works, all this UI stuff gets me a bit confused.